src/_posts/compliance/security-overview/2000-01-01-physical-security.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Physical Security
+nav: Physical
+modified_at: 2024-06-20 00:00:00
+tags: compliance security measures
+index: 1
+---
+
+### Datacenters
+
+Scalingo uses datacenters that are certified ISO 27001, ISO 27017, ISO 27018, HDS (Region `osc-secnum-fr1`) and
+SecNumCloud (Region `osc-secnum-fr1`). These certifications ensure that the datacenters have implemented a high level of
+security to protect the data hosted on them.
+
```

Comment:
For everything below in Physical section: be careful to not act anything on behalf of Outscale.
We may just want to say our Host Provider is following the best industry standards for security, and if the customer need more information, they should directly reach them or ask through our support team

Reply:
Are you sure ? We have this information from Outscale

Reply:
We may want to keep these information written down here, but it must be clear that it's related to our N-1 provider no?

Reply:
Fixed